scripts/common.sh.in

```diff
@@ -371,3 +371,28 @@ check_minion_id() {
         return 1
     fi
 }
+
+rotate_logfile() {
+    # because set -u, we pass an empty string as default value
+    _rotate_logfile ""
+}
+
+_rotate_logfile() {
+    # Rotate logfile if it exists
+    LOGFILE_MAX_ROTATIONS=10
```

Comment:
nitpick: Can you move this variable definition at the very beginning of the `common.sh` script (with the other ones)